### DISCUSSION TITLE
Is there a way to select a default timeframe (or change it for all sites at once) on the dashboard?

### DISCUSSION THREAD

**Discussion by jpvalery** [umami-software/umami#598] [Ideas]
Hey there,

Hope everyone's doing alright.

I'm more interested in a different timeframe than "Last 24 hours" on my Umami dashboard. So i'm left doing multiple clicks for each site to change it to "Last 30 days" for instance.

Wouldn't it be nice to be able to select a default in the settings or at least change the timeframe for all site at once?

Just a thought :)

**Comment by mikecao** [comment]
Settings -> Profile -> Default date range

**Reply by jpvalery** [reply]
🤦 don't know how I missed that. Thanks for the fast reply!
